bowtie/_core.py

```diff
@@ -714,12 +714,13 @@ def syntax(self) -> RenderableType:
     @classmethod
     def from_dict(
         cls,
+        instance: Any = None,
```

Comment:
I'm not a fan of this -- I assume you mean to make this optional (by `None`)? But that's extra icky because `None` is a perfectly valid instance. I'll try to see where you're relying on this but it would seem to suggest another type would be better for whatever you're using this for.

Reply:
I am not exactly able to recall why I added this :( . It was probably the None instance failing bowtie run or something in some of the benchmark. It was raising the error instance cannot be left empty.